python_modules/dagster/dagster/__init__.py

```diff
@@ -284,7 +286,9 @@
     TableSchemaMetadataValue as TableSchemaMetadataValue,
     TextMetadataValue as TextMetadataValue,
     TimestampMetadataValue as TimestampMetadataValue,
+    UrlCodeReference as UrlCodeReference,
     UrlMetadataValue as UrlMetadataValue,
+    link_to_source_control as link_to_source_control,
     with_source_code_references as with_source_code_references,
```

Comment:
I didn't realize `with_source_code_references` was a top-level export. Has this shipped already?

Reply:
No, it'll go out with next week's release, so we can definitely roll this back.

Reply:
Will drop the top-level export for now.